app/assets/stylesheets/maplibre-gl-all.scss

```diff
@@ -0,0 +1,6 @@
+
+/*
+ *= require maplibre-gl
+ */
+
+/* BASED ON leaflet-all.scss */
```

Comment:
This comment probably can go. (I see it's like this in OHM.)

Reply:
```suggestion
```